site/content/device/MF-RC522

```diff
@@ -1,7 +1,7 @@
 +++
```

Comment:
Please name the file `mf-rc522.md` as all the other files are lower case, and it won't work without a .md file extension. Thanks!

Reply:
Ok, changed the name.